Add ConfigCat provider to Feature Flag Data in RUM

endret opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
Add a new feature flag provider [ConfigCat](https://configcat.com) to the _Getting Started with Feature Flag Data_ documentation.


### Merge instructions
<!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->

- [X] Please merge after reviewing

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

drichards-87 commented:
Created DOCS-8136 for editorial review.

drichards-87 commented:
Thanks for submitting this PR @endret. I've created a Jira card for the Docs Team to track the editorial review. Someone on the team will review it, and get back to you if they have any follow-up questions. Thanks!

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
```diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+{{< tabs >}}
+{{% tab "Browser" %}}
+
+Initialize the ConfigCat Javascript SDK with subscribing to the `flagEvaluated` event that will report feature flag evaluations to Datadog:
```
```suggestion
Initialize the ConfigCat Javascript SDK with subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

endret replied:
I also applied `by` instead of `with` to ensure consist texts.

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
````diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+  );
+```
+
+For more information about initializing the ConfigCat Javascript SDK, see [Documentation](https://configcat.com/docs/sdk-reference/js).
````
```suggestion
For more information about initializing the ConfigCat Javascript SDK, see ConfigCat's [JavaScript SDK documentation][1].

[1]: https://configcat.com/docs/sdk-reference/js
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
```diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+Initialize the ConfigCat Swift (iOS) SDK with subscribing to the `flagEvaluated` event that will report feature flag evaluations to Datadog:
```
```suggestion
Initialize the ConfigCat Swift iOS SDK by subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
````diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+  }
+```
+
+For more information about initializing the ConfigCat Swift (iOS) SDK, see [Documentation](https://configcat.com/docs/sdk-reference/ios).
````
```suggestion
For more information about initializing the ConfigCat Swift (iOS) SDK, see ConfigCat's[Swift iOS SDK documentation][1].

[1]: https://configcat.com/docs/sdk-reference/ios
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
```diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+{{% /tab %}}
+{{% tab "Android" %}}
+
+Initialize the ConfigCat Android SDK with subscribing to the `flagEvaluated` event that will report feature flag evaluations to Datadog:
```
```suggestion
Initialize the ConfigCat Android SDK by subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
````diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+  });
+```
+
+For more information about initializing the ConfigCat Android SDK, see [Documentation](https://configcat.com/docs/sdk-reference/android).
````
```suggestion
For more information about initializing the ConfigCat Android SDK, see ConfigCat's [Android SDK documentation][1].

[1]: https://configcat.com/docs/sdk-reference/android
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
```diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+{{% /tab %}}
+{{% tab "Flutter" %}}
+
+Initialize the ConfigCat Dart SDK with subscribing to the `flagEvaluated` event that will report feature flag evaluations to Datadog:
```
```suggestion
Initialize the ConfigCat Dart SDK by subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
````diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+  );
+```
+
+For more information about initializing the ConfigCat Dart SDK, see [Documentation](https://configcat.com/docs/sdk-reference/dart).
````
```suggestion
For more information about initializing the ConfigCat Dart (Flutter) SDK, see ConfigCat's [Dart SDK documentation][1].

[1]: https://configcat.com/docs/sdk-reference/dart
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
```diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+
+{{% tab "React Native" %}}
+
+Initialize the ConfigCat React SDK with subscribing to the `flagEvaluated` event that will report feature flag evaluations to Datadog:
```
```suggestion
Initialize the ConfigCat React SDK by subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
````diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+</ConfigCatProvider>
+```
+
+For more information about initializing the ConfigCat Dart SDK, see [Documentation](https://configcat.com/docs/sdk-reference/react).
````
```suggestion
For more information about initializing the ConfigCat React SDK, see ConfigCat's [React SDK documentation][1].

[1]: https://configcat.com/docs/sdk-reference/react
```

rtrieu reviewed (changes requested):
Hi, thanks @endret for your contribution! i've left some feedback to adhere to our team's style guide. I updated the integration logo so that it is properly centered.

rtrieu commented on layouts/partials/rum/rum-feature-flag-tracking.html:
```diff
@@ -11,6 +11,15 @@
+      <a class="card h-100"
+        href="/real_user_monitoring/guide/setup-feature-flag-data-collection/?tab=browser#configcat-integration">
+        <div class="card-body text-center py-2 px-1">
+          {{ partial "img.html" (dict "root" . "src" "integrations_logos/configcat_horiz.svg" "class" "img-fluid" "alt"
```
```suggestion
          {{ partial "img.html" (dict "root" . "src" "integrations_logos/configcat_large.svg" "class" "img-fluid" "alt"
```

endret commented:
> Hi, thanks @endret for your contribution! i've left some feedback to adhere to our team's style guide. I updated the integration logo so that it is properly centered.

Thanks to reviewed it, in this case I removed the logo from this PR and applied all changes.

rtrieu commented on content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:
````diff
@@ -195,6 +195,122 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
+  }
+```
+
+For more information about initializing the ConfigCat Swift (iOS) SDK, ConfigCat's[Swift iOS SDK documentation][1].
````
```suggestion
For more information about initializing the ConfigCat Swift (iOS) SDK, see ConfigCat's[Swift iOS SDK documentation][1].
```

rtrieu reviewed (changes requested):
One small fix.

rtrieu commented:
Hi @endret , just letting you know the linked PR was merged today so the final copy will appear on the docs site in about 20-30 minutes. Thanks again for your contribution!
